hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1936,7 +1938,12 @@ public String listStatus(final Path path, final String startFrom,
           );
           perfInfo.registerResult(op.getResult());
           BlobList blobList = op.getResult().getBlobList();
+          LOG.debug("List Blob Call on filesystem: {} path: {} marker: {} delimiter: {} returned {} objects",
+              client.getFileSystem(), prefix, continuation,
+              delimiter, blobList.getBlobPropertyList().size());
+
           continuation = blobList.getNextMarker();
+          objectCountReturnedByServer += blobList.getBlobPropertyList().size();
```

Comment:
Where are we adding it as a parameter to the tracing context ?

Reply:
blobList.getBlobPropertyList().size() can be stored in a variable as it is used on line 1943 and 1946 both

Reply:
We are not adding this data in tracing context as of now.
We will be getting this data only after we are done with the network call. I think it does not make sense to add this data into tracing context for next network call.

Let me know your thoughts here...